### DISCUSSION TITLE
When will struts 1.5 be released?

### DISCUSSION THREAD

**Discussion by jesa43** [ste-gr/struts1-reloaded#38] [General]
excited, but when will the officiell version of 1.5 be released?

**Comment by ste-gr** [comment]
Hi @jesa43,

the current version 1.5-RC2 should be useable. Currently their is an open issue (#32). Here I'll wait for response.
The "main-blocker" for 1.5 is the missing generation of the taglib-docs.

Greetings
Stefan

**Comment by ste-gr** [comment]
duplicate #47
